packages/core/aqua-to-js/package.json

```diff
@@ -14,16 +14,16 @@
   "keywords": [],
   "author": "Fluence Labs",
   "license": "Apache-2.0",
-  "dependencies": {
-    "ts-pattern": "5.0.5"
-  },
   "devDependencies": {
     "@fluencelabs/aqua-api": "0.12.0",
     "@fluencelabs/aqua-lib": "0.7.3",
     "@fluencelabs/interfaces": "workspace:*",
+    "@fluencelabs/js-client": "workspace:^",
     "@fluencelabs/registry": "0.8.7",
     "@fluencelabs/spell": "0.5.20",
     "@fluencelabs/trust-graph": "0.4.7",
-    "vitest": "0.34.6"
+    "ts-pattern": "5.0.5",
```

Comment:
Is `packages/core/aqua-to-js/src/common.ts` not part of the actual lib? If it is - then `ts-pattern` seems to be neeeded in `dependencies` section right because it is imported in `packages/core/aqua-to-js/src/common.ts`?

Reply:
Fixed